packages/ra-core/src/reducer/admin/resource/list/initiated.js

```diff
@@ -0,0 +1,12 @@
+import { CRUD_GET_LIST_SUCCESS } from '../../../../actions/dataActions';
+export default resource => (previousState = false, { type, meta }) => {
```

Comment:
how about using the `total` reducer instead (modifying it to start with a `null` state instead of zero)?

Reply:
I was actually looking for one to use, and opted out from `ids` because a list can be empty, but didn't notice total, which is a good place to do this 👍